.storybook/Container.js

```diff
@@ -12,7 +12,7 @@ export default class Container extends Component {
           padding: '3rem',
           display: 'flex',
           flexDirection: 'column',
-          alignItems: 'center',
+          alignItems: 'stretch',
```

Comment:
Does this conflict with the new wrapper component that some stories use to be full width? I think we want the smaller components to still be centered.